src/test/playwright/playwright.config.ts

```
@@ -30,10 +30,20 @@ export default defineConfig({
         ignoreHTTPSErrors: true,
     },
 
-    /* Configure projects for major browsers */
+    /* Configure projects for fast and slow tests */
     projects: [
+        // Tests with @slow tag
         {
-            name: 'chromium',
+            name: 'slow-tests',
+            grep: new RegExp('@slow'),
```

Biome

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)